Homework-2-In-Memory-CRUD-REST-Service/src/controllers/userController.js

```diff
@@ -0,0 +1,58 @@
+import { userFiltersExist } from '../helpers/usersHelper.js';
+import * as UserService from '../services/userService.js';
+
+export const getAllUsers = (req, res) => {
+    const filters = req.query;
+    let users;
+
+    if (userFiltersExist(filters)) {
```

Comment:
I'm not sure if there is much sense in having separate methods for getting all users and for getting all users but applying filters on top of it.

Reply:
Yeah, agree! It's redundant and will be fixed!